.travis.yml

```diff
@@ -5,6 +5,10 @@ python:
 env:
     - SHOTGUN_EXTERNAL_REPOS_ROOT=/tmp
 
+dist: xenial
```

Comment:
Submit this back to Shotgun, adding the URL for official Travis Docs

Remove the commented out bits later on in this file